@xen-orchestra/rest-api/src/xoa/xoa.controller.mts

```diff
@@ -44,4 +44,13 @@ export class XoaController extends Controller {
       return dashboard
     }
   }
+
+  @Get('xo-versions')
```

Comment:
I'm not a fan of the endpoint name, as it can be confusing with `xo-server version, XOA version`.

maybe `xo-urls`? 
/cc @pdonias Do you have a better idea?